src/main/java/frc/robot/maps/GastonMap.java

```diff
@@ -16,6 +16,25 @@
 import edu.wpi.first.wpilibj.interfaces.Gyro;
 
 public class GastonMap extends RobotMap {
+    // Climber
+    private final DigitalInput leftUpperLimit = new DigitalInput(0);
+    private final DigitalInput leftLowerLimit = new DigitalInput(1);
+    private final DigitalInput rightUpperLimit = new DigitalInput(2);
+    private final DigitalInput rightLowerLimit = new DigitalInput(3);
+
+    private final PIDSparkMax leftMotor = new PIDSparkMax(0, MotorType.kBrushless);
+    private final PIDSparkMax rightMotor = new PIDSparkMax(1, MotorType.kBrushless);
```

Comment:
These should be declared within the `getLeftTelescopeMap` and its right equivalent.